Dockerfile

```diff
@@ -1,17 +1,17 @@
 # Build the manager binary
-FROM golang:1.16 as builder
+FROM registry.access.redhat.com/ubi8/go-toolset:latest as builder
```

Comment:
Shouldn't this be similar (if not identical) to the way we build (for example) the cluster-ingress-operator?

```
FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.16-openshift-4.10 AS builder
```

Taken from: https://github.com/openshift/cluster-ingress-operator/commit/c22568aac5be683cd0dbf1eba43522a9645455ca

Reply:
This Dockerfile is for end-users building manually (e.g. in the README.md manual steps for "make image").

In this repo, there is Dockerfile.openshift which tracks that change for the 4.9 ... 4.10 changes, and which uses the ART images.